libraries/STM32F0xx_StdPeriph_Driver/src/stm32f0xx_interrupt.c

```diff
@@ -92,3 +92,17 @@ StatusCode stm32f0xx_interrupt_exti_clear_pending(uint8_t line) {
   EXTI_ClearITPendingBit((uint32_t)0x01 << line);
   return STATUS_CODE_OK;
 }
+
+StatusCode stm32f0xx_interrupt_exti_mask_set(uint8_t line, bool masked) {
+  if (line >= 16) {
```

Comment:
You should be comparing `line` to `NUM_STM32F0XX_INTERRUPT_LINES` instead of 16 - since this is a generic (non-GPIO) function, we should support masking other interrupt lines as well.